test/library/packages/UnitTest/PopulateTests_PRIM/TestCountClass.good

```diff
@@ -0,0 +1 @@
+0
```

Comment:
We could make this a semantic future if we thought that behavior might change as we expand the feature.  That would involve changing this to `1`, adding a `.bad` file with `0` as the count, and adding a `.future` that references the issue where we talked about if class methods should be considered tests.

Reply:
Added.